app/screens/login/index.tsx

```diff
@@ -25,13 +25,17 @@ import ErrorText, {ClientErrorWithIntl} from '@components/error_text';
 import {FORGOT_PASSWORD, MFA} from '@constants/screens';
 import FormattedText from '@components/formatted_text';
 import {useManagedConfig} from '@mattermost/react-native-emm';
-import {scheduleExpiredNotification} from '@requests/remote/push_notification';
-import {login} from '@requests/remote/user';
+import {scheduleExpiredNotification} from '@actions/remote/push_notification';
+import {login} from '@actions/remote/user';
 import {goToScreen, resetToChannel} from '@screens/navigation';
 import {t} from '@utils/i18n';
 import {preventDoubleTap} from '@utils/tap';
 import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
 
+
+// TODO: invalidate client when returning back to the select server screen
```

Comment:
No need, this should be done when login fails, for the ping and getting the unauthenticated config & license we just need the generic client on the server screen